packages/apsara-ui/src/Tooltip/Tooltip.tsx

```diff
@@ -19,11 +20,18 @@ const Tooltip = ({
     style = { backgroundColor: "#333", color: "white" },
     children,
     defaultOpen = false,
+    tooltipOpen = false,
     ...props
 }: TooltipProps) => {
+    const [open, setOpen] = React.useState(false);
```

Comment:
do we need state here?

can we just pass open from prop to the library?

Reply:
We need to use open in conjunction with onOpenChange or else the tooltip won't close.
I tried to move this state to the code where we use this library but it was creating issues.

Reply:
removed the state from here, it can be now handled by the component or code which is using the library.